go-controller/pkg/ovn/pods.go

```diff
@@ -231,6 +231,11 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) error {
 	var out, stderr string
 	var err error
 
+	//check if the node has a hostsubnet assigned to it
```

Comment:
@JacobTanenbaum can you clarify this comment to something more like "don't wait for the logical switch to appear if the node has no assigned hostsubnet" or something like that?

Reply:
fixed